Tests/DeepLink/DPLDeepLink_AppLinksSpec.m

```diff
@@ -1,5 +1,8 @@
-@import DeepLinkKit.Private;
-@import DeepLinkKit.AppLinks;
+#import "DPLDeepLink.h"
```

Comment:
What prevents you importing the module here? This will still work externally, right?